packages/components/types/components/Alert.d.ts

```diff
@@ -41,7 +41,7 @@ export declare type AlertProps = {
  * </Alert>
  * ```
  */
-export declare const Alert: PolymorphicComponent<AlertProps>;
+export declare const Alert: PolymorphicComponent<'div', AlertProps>;
```

Comment:
This parameter order changed and it is now intentionally required (to avoid mistakes). This is clearer when we're typing the component next to the code rather than in definition files as it'll be more obvious what the box is for each component.